sampler_t.adoc

```diff
@@ -46,7 +46,7 @@ The sampler fields are described in the table below:
 
 |
 
-Specifies whether the `x`, `y` and `z` coordinates are passed in as normalized or unnormalized values.
+Specifies whether the `x`, `y` and `z` coordinates are passed in as normalized ([0,1[) or unnormalized ({0, size-1}) values.
```

Comment:
>This led me to the conclusion that 1.0 already gives an out of border value and therefore the right side is not included (hence `[`).

Oh, I did not know that `[` means that the right side is not included. I always use ')' to express that. In that case it's ok. 

>`[]` because by using normalised coordinates all continuous values between 0 and 1 are possible and `{}` because only the discrete values `{0,1,2,...,size-1}` are valid.

Right. I would replace  `{0, size-1}`  with `{0, ..., size-1}`, so it's easier to understand. Are you sure they must be integers?

I was able to find that it works the other way around: if you use integer coordinates sampler must be set in `CLK_NORMALIZED_COORDS_FALSE` mode.

>The read_imagef calls that take integer coordinates must use a sampler with filter mode set to
CLK_FILTER_NEAREST, normalized coordinates set to CLK_NORMALIZED_COORDS_FALSE and addressing mode set to CLK_ADDRESS_CLAMP_TO_EDGE, CLK_ADDRESS_CLAMP or CLK_ADDRESS_NONE; otherwise the values returned are undefined

On the other hand, if you don't use normalized coords, then you probably want to use integer coordinates (`{0, ..., size-1}`).